terraso_backend/apps/project_management/models/sites.py

```diff
@@ -53,15 +52,6 @@ class Meta(BaseModel.Meta):
         verbose_name="owner to which the site belongs",
     )
 
-    created_by = models.ForeignKey(
```

Comment:
Instead of using `created_by`, the site can be owned by a person (`owner` field) or a project (`project`). Basically the created_by was changed to owner to be a bit clearer, and it slipped back in by accident. I had to remove it here to fix up some tests.